dace/transformation/dataflow/strip_mining.py

```diff
@@ -125,6 +125,7 @@ def calc_set_union(set_a, set_b):
 class StripMining(transformation.SingleStateTransformation):
     """ Implements the strip-mining transformation.
 
+        TODO: Update doc
```

Comment:
Please do

Reply:
As discussed it is a TODO from long ago + we got no unit tests for strip mining or map tiling